jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -948,6 +948,11 @@ YUI.add('juju-controller-api', function(Y) {
             return prev;
           }
           prev[name] = this._parseCloudResult(result.cloud);
+          if (!prev['.names'] || !prev['.names'].length) {
```

Comment:
May have been left over due to scoping issues we were running into in the debugger (I want to get `data[name]`, but `name` didn't appear available in the inner arrow func, so I worked around it). I'll research if that's still needed.